### ISSUE TITLE
Adapt the Wordpress exporter for Ghost 1.0 and Ghost 2.0

### ISSUE THREAD

**Issue by kirrg001** [TryGhost/wp-ghost-exporter#12]
### Meta Version

The wordpress ghost exporter currently exports the following static meta version: `000`

e.g.

```
{"meta":{"exported_on":"...","version":"000"}}
```

Ghost **0.3.0** has used this database version (this is 5 years ago).

In Ghost 1.0 we have removed this database version notation completely. That means, the exporter no longer exports this version notation, instead it exports the the current Ghost version you are on.

e.g.

```
{"meta":{"exported_on":...,"version":"1.25.4"}}
```

The Ghost team is currently [preparing for Ghost 2.0](https://github.com/TryGhost/Ghost/issues/9742) and this major release will disallow importing 0.X Ghost exports. We only support the current and latest major (Ghost 1.0 and Ghost 2.0 in this case). 

**If we don't change the meta version field in the wordpress exporter, exports won't import into Ghost 2.0.**

Ideas?

- introduce `meta.version=wp-0.5.6` (wp as prefix and the wp exporter version you are using)

### Check compatibility

Furthermore we should look into the compatibility in general. Many fields were renamed in Ghost 1.0 e.g. image fields.

**If we don't adapt the field names in the wordpress exporter, exports won't correctly import into Ghost 2.0.**

**Comment by hugoprudente** [comment]
Hi, is this the structural pattern that should be used?

https://docs.ghost.org/api/migration/

Any further consideration that I should be mindful?

A friend has a blog and I'm migrating 7 years of data to ghost, so I'll need to work on the plug-in

**Comment by ErisDS** [comment]
@hprudent I'd love to talk to you about your use case in more detail, could you move the convo to forum.ghost.org? I'm _actively_ working on new, modern, better tooling for migrations and I think I can help a lot.

**Comment by nuclearpengy** [comment]
hi @ErisDS - is there an active thread about this on the forum?

**Comment by nuclearpengy** [comment]
This should be sorted now.

**Comment by nuclearpengy** [comment]
This issue has been addressed with the release of version 1.0.0, available for download here: https://wordpress.org/plugins/ghost/

Closing.
